doc/api/quic.md

```diff
@@ -1246,12 +1246,16 @@ empty object when the key exchange is not ephemeral. The supported types are
 
 For example: `{ type: 'ECDH', name: 'prime256v1', size: 256 }`.
 
-#### `quicclientsession.setSocket(socket])`
+#### `quicclientsession.setSocket(socket[, natRebinding])`
```

Comment:
```suggestion
#### `quicclientsession.setSocket(socket\[, natRebinding\])`
```

Reply:
This was actually from a preceeding PR. Will open another to address this